README.md

```diff
@@ -299,34 +338,144 @@ Retrieving a single page returns a [PageResponse&lt;T&gt;](#page-response-class)
 | ---|---|
 |[PagesObjectMismatchException](#pagesobjectmismatchexception)|This exception will be thrown when the library can't fit the returned data into the passed object class. |
 
-#### Examples:
+#### Examples
+
+##### Controller and ViewModel examples
+
 ```C#
-PageResponse<ProductPage> saleOfTheDayPage = butterClient.RetrievePage<ProductPage>("products", "saleoftheday");
 
-var paramterDict = new Dictionary<string, string>() 
+public namespace HungryDevApp
 {
-    {"preview", "1"},
-};
-PageResponse<ExamplePage> stuffedArtichokesPage = await butterClient.RetrievePageAsync<ExamplePage>("recipes", "stuffed-artichokes", paramterDict);
+    public class RecipesController : Controller
+    {
+        [Route(recipe/{slug})]
+        public virtual ActionResult Index(string slug)
+        {
+            var butterClient = new ButterCMSClient("API KEY");
+
+            PageResponse<ProductPage> recipe = butterClient.RetrievePage<ProductPage>("recipe", slug);
```

Comment:
Should ProductPage be RecipePage?

Reply:
Yep, copy and paste error. It's been fixed with the latest commit